.github/workflows/build_and_test.yaml

```diff
@@ -119,6 +121,8 @@ jobs:
           echo -n "This absolutely exists" | diff - /tmp/absolute.txt &&
           echo -n "This relatively exists" | diff - /tmp/relative.txt &&
           [ ! -f /tmp/500.txt ] && echo "No 5xx file created" &&
+          echo -n "This generated by script" | diff - /tmp/command-output.txt &&
```

Comment:
Given the example script this assertions is likely to fail (as the script generates random hex text, doesn't it?)